### PR TITLE
CPCN-145

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

File: assets/company-admin/components/CompanyAdminApp.jsx [hunk 1]
```diff
@@ -149,8 +149,8 @@ class CompanyAdminAppComponent extends React.Component {
                                 </button>
                             </div>
 
-                            {this.props.sectionId === 'users' ? <label className='label label--big label--rounded'>{
-                                gettext('Users : {{total}}', {total : this.props.totalUsers})}</label> : ''}
+                            {this.props.sectionId === 'users' ? <label className='label label--big label--rounded me-2'>{
+                                gettext('UTILISATEURS : {{total}}', {total : this.props.totalUsers})}</label> : ''}
```
Comment (petrjasek) [inline assets/company-admin/components/CompanyAdminApp.jsx:153]: this should be again `'Users'` in the gettext

File: assets/company-admin/components/CompanyAdminApp.jsx [hunk 2]
```diff
@@ -117,7 +117,7 @@ class CompanyAdminAppComponent extends React.Component {
                         >
                             <button
```
Comment (MarkLark86) [inline assets/company-admin/components/CompanyAdminApp.jsx:118]: @petrjasek Should we comment out this button? At the moment we have the CompanyAdmin sidebar with 1 entry. Removing the button now until we add more pages to CompanyAdmin?

Comment (petrjasek) [reply assets/company-admin/components/CompanyAdminApp.jsx:118]: yep that would make sense

File: assets/company-admin/components/CompanyAdminApp.jsx (hunk 1 above)
Comment (darconny) [reply assets/company-admin/components/CompanyAdminApp.jsx]: done

File: assets/company-admin/components/CompanyAdminApp.jsx (hunk 2 above)
Comment (darconny) [reply assets/company-admin/components/CompanyAdminApp.jsx]: done

**Pull request merged**